lib/mocha/invocation.rb

```diff
@@ -53,7 +55,9 @@ def arguments
     end
 
     def call_description
-      "#{@mock.mocha_inspect}.#{@method_name}#{ParametersMatcher.new(@arguments).mocha_inspect}"
+      description = "#{@mock.mocha_inspect}.#{@method_name}#{ParametersMatcher.new(@arguments).mocha_inspect}"
+      description << ' { ... }' unless @block.nil?
```

Comment:
We could make this more informative as below:

1. { |arg1, arg2| <block body> } relying on Proc#arity
2. { |x, y, *other| <block body> } relying on Proc#parameters

2 would be smarter but more complex. 1 might be a good enough solution. I expect call_description to be a rarely used feature anyway. So, I'd avoid being smart at the cost of complexity until a need is validated.

Even the current form might be good enough.

In any case, I suggest replacing ... with something like <block body>, making it clearer that we're representing a block.

Reply:
Hmm. Thanks. I did consider doing something with `Proc#arity`. However, I think what I have is good enough for now and I don't think anyone should start relying on the exact format, so we should be free to change it at a later date.

I hadn't come across `Proc#parameters`. That's a nice addition to the language, but I assume it's only been introduced relatively recently...?

Reply:
I'm going to stick with what I've got for now, i.e. "{ ... }".